src/api/spec/db/data/backfill_notified_projects_spec.rb

```diff
@@ -39,13 +39,16 @@
     end
 
     it 'backfills the notifications_projects table with all projects from existing notifications' do
-      expect(NotifiedProject.pluck(:project_id)).to eq([
+      expected = NotifiedProject.pluck(:project_id).sort
+      real = [
         bs_request_with_submit_action.target_project_objects.distinct.map(&:id),
         user_review.bs_request.target_project_objects.distinct.map(&:id),
         comment_project.commentable.id,
         comment_package.commentable.project_id,
         comment_request.commentable.target_project_objects.distinct.map(&:id)
-      ].flatten!)
+      ].flatten!.sort
+
+      expect(expected).to eq(real)
```

Comment:
```suggestion
      ].flatten!

      expect(expected).to contain_exactly(real)
```
Here we could use the [contain_exactly](https://rspec.info/features/3-12/rspec-expectations/built-in-matchers/contain-exactly/) matcher, so there is no need to use `.sort`.